.github/workflows/ci.yml

```
@@ -0,0 +1,62 @@
+name: CI
+'on':
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+jobs:
+  # lint-dockerfile:
```

As in https://github.com/telefonicaid/fiware-sth-comet/pull/550, I'd suggest:

```suggestion
  # FIXME: hadolint won't work since the docker file has not been updated.
  # lint-dockerfile:
```